play-json/jvm/src/test/scala/play/api/libs/json/JsonValidSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.api.libs.json
+
+import org.specs2.mutable._
+import play.api.libs.json._
+import play.api.libs.json.Json._
+import play.api.libs.functional.syntax._
+
+class JsonValidSpec extends Specification {
+  "JSON reads" should {
```

Comment:
Part of the origin `JsonValidSpec` specific to the JVM backend (no regression for this case)